### ISSUE TITLE
Problem compiling the latest version of Potlach

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#2842]
**[Submitted to the original trac issue database at 2.34pm, Monday, 29th March 2010]**

I get the 4 or 5 of the following errors when trying to compile the latest version:

```
applications/editors/potlatch2/net/systemeD/halcyon/Map.as(250): col: 40 Error: Type was not found or was not a compile-time constant: EntityRenumberedEvent.

                private function wayRenumbered(event:EntityRenumberedEvent):void {
```

Looking at the code it was added 19-March.

So looks like the associated file was not added to accompany that commit.

**Comment by tomhughes** [comment]
**Author: richard**
**[Added to the original trac issue at 2.45pm, Monday, 29th March 2010]**

(In [20733]) add missing file. Fixes #2842

**Comment by tomhughes** [comment]
**Author: Richard**
**[Added to the original trac issue at 2.46pm, Monday, 29th March 2010]**

I really do hate Subversion.
